service/models.py

```diff
@@ -79,7 +79,7 @@ def serialize(self):
         return {
             "product_id": self.product_id,
             "name": self.name,
-            "condition": self.condition.value,
+            "condition": self.condition and self.condition.value,
```

Comment:
why do we have an and here?

Reply:
It was a temporary fix to an issue, forgot to remove. Thanks, pushing an update.

Reply:
okie